### DISCUSSION TITLE
How to get the topic of client (subscriber) via RabbitMQ Management when using wildcards for routing key in definition.json file?

### DISCUSSION THREAD

**Discussion by chienhust** [rabbitmq/rabbitmq-server#8097] [Questions] [answered]
- In definition.json file of RabbitMQ Management, i set wildcards like : "routing_key": "data.#.device.#.usecase".
- In topic of client, I publish messages to "data.12345.device.6789.usecase" topic. On RabbitMQ Management, i can see the messages are recieved via http://localhost:15672/#/queues.
How to get the topic of client (subscriber) via RabbitMQ Management (http://localhost:15672/api/)?
Thank you very much.

**Comment by michaelklishin** [comment, marked as answer]
You cannot see actual routing keys of published messages in the management UI.

Consumers can observe them by inspecting the delivery metadata (it's not the same thing as message properties but some clients will lump them together into a single data structure).

You can also see them in the RabbitMQ Firehose log.

**Reply by chienhust** [reply]
I can see actual routing keys via log file of RabbitMQ Firehose.
Thank you very much.
